app/decorators/site_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class SiteDecorator < Draper::Decorator
+  delegate_all
+
+  def full_address
```

Comment:
Do we really need the decorator for this?

Reply:
Probably not but I don't think it should go into the model either as it's mainly for presentational purposes. Will leave it for now but happy to bin it later when we clean things up and after we have an alternative.